docs/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Can you put it in the .gitignore in the root directory? Let's only have one gitignore

Reply:
Resolved